CS-Capstone-Team-4-Terawe/GeoNewsFinder4
# Added google maps for iOS and Android

Added google maps interface for iOS and Android.

To test:
1. Cd into GeoNewsFinder4
2. Run "npm install"
3. Download Expo Go from app store or google play store
4. Run "npx expo start"
5. Scan QR code with device

Map should be interactive. Users can zoom in/out and scroll to different locations.
